config/locales/cy/cy.yml

```diff
@@ -71,4 +71,42 @@ cy:
       password:
         label: Crëwch eich gair cofiadwy
         hint: "Os ydych wedi nodi eich cyfeiriad e-bost uchod, byddwch yn gallu ailosod eich gair cofiadwy os byddwch yn ei anghofio"
+  activemodel:
```

Comment:
I think you have put this in an inconsistent file when compared with the English translation file for active model errors (config/locales/en/et1/active_model.en.yml)